data/hai/culture hai.txt

```diff
@@ -0,0 +1,208 @@
+			choice
+				`	"Sure, can I just get some food here?"`
+					goto wrapup
+				`	"Whats the Musical Entertainment Expo?"`
+				`	"Why don't they start drinking yet?"`
+			`	"Well you see the big music labels do a lot of talent scouting all year round, they all have a department that is solely dedicated to finding and assembling new music groups to be the next big thing. It's traditionally to celebrate only after you've committed to the journey with a new band. Every three years the industry holds ME2, that's what you're seeing up there, to debut their next round of bands, and to tease any mid-season releases they might have in the works."`
+			choice
+				`	"Your music comes off a production line?"`
+					goto production
+				`	"Do you not have musicians trying to make it on their own?"` 
+			`	"Oh for sure, but mostly just to get picked up by one of these companies."`
+			label production
+			`	The bartender chuckles. "We've been doing pop music for longer than your people have had fire, it's about the experience of being a fan of a group as it rises to fame and stardom. A dedicated Hai can go back and find a song for almost any possible life experience in almost any possible genre. It's the journey, and the personalities, that are what it's all about."`
+			choice
+				`	"How strange."`
+				`	"How interesting."`
+			`	"Anyway, I gather you were you after some food?"`
+			label wrapup
+			`	The bartender tips his head in your direction and takes your order. "No problem, it'll be right up."`
+			`	It's a noisy meal.`
+				decline
+
+#This uses the flag from receiving the Hai translator.
+mission "ZZCU The Meme Wars 1"
+	minor
+	source
+		government "Hai"
+		not attributes "station"
+	to offer
+		random == 1
+	on offer
+		conversation
+			`Stepping into the spaceport you are beset by a riot of color. Every blank surface that's not usually paid advertising space seems to be festooned with dangling holographic projectors, ribbons, banners, and even actual paper fliers and posters, apparently taped to whatever surface will bear them. At first you wonder if there's some sort of festival going on, but as you watch a young Hai wearing a hood slaps up another holo projector which immediately projects the image of an anthro root vegetable with a sword hacking at some kind of impressionist concept of a Hai elder made up of iridescent triangles. The young Hai then proceeds to tear down a banner and slap some goo on a massive poster which begins dissolving before a pair of security guards, wearing some very obviously non-standard additions to their uniforms, chase him back out of the spaceport.`
+			branch translator
+				has "language: Wanderer"
+			`	As you watch the triangle elder being holoprojected seems to flicker and then fizzles out as the battery unit fades, apparently under the assault of the root vegetable which appears to do a victory dance, before proceeding to act out something that makes precisely no sense to you.`
```

Comment:
Rewrite this from the ground up, it makes no sense.

Reply:
I assume you're referring to line 86, it's an e-warfare device inside a holoprojector. When the projected memes do battle the underlying computers are also trying to kill each other. One wins.

Reply:
I mean that it's written and conveyed poorly.

Reply:
It has edited based on vkr's feedback, so maybe look there, but beyond that I'm afraid unless you can be more specific I can't see particularly what the complaint is.